packages/gatsby/src/redux/persist.ts

```diff
@@ -13,6 +13,9 @@ import { IGatsbyNode, ICachedReduxState } from "./types"
 import { sync as globSync } from "glob"
 import report from "gatsby-cli/lib/reporter"
 
+let warnedAboutChunkSize = false
+let warnedAboutChunkGuess = 0 // 0, 1, 2
```

Comment:
These are required because the cache is stored in three distinct steps and we don't want to clutter the output with flag related messages more than once per session.